tools/pactlinfo.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+# Licensed under GPLv3.
+
+import re
+import os
+import subprocess
+import json
+
+# parse_pactl_list() is modified from the file _audio_data.py
+# in the package apport-symptoms
+def parse_pactl_list(pactlvalues):
```

Comment:
A simple question here. If we have a help function, why still need `|` and jq for the string process? Why not do something like `sof-dump-status.py` to use args to directly output info you need.